bridge-core/dash-compiler
# Resolve an os error if the "components" folder did not exist but the plugin was in the config

# Summary
Added a catch to the check for files in the components folder to prevent an error if the folder does not exist but the custom component compiler plugins are enabled.  

Resolves an issue related to #2 which resolved bridge-core/editor#866 & bridge-core/editor#731 but introduced an issue where if the compiler plugins are enabled but no components folder exists os error 3 occurs.

# To Test
- Create a bridge project 
- Confirm it builds and watch mode functions
- Add `"customEntityComponents"` , `"customItemComponents"` , `"customBlockComponents"` to project config.
- Confirm that os error 3 does not occur 
- Confirm that building and watch mode still work
- Add a custom component, such as an entity component
- Confirm that it builds correctly 
- Confirm that watch mode does not remove the custom component
